cli/src/semgrep/metrics.py

```diff
@@ -62,6 +62,14 @@
 
 METRICS_ENDPOINT = "https://metrics.semgrep.dev"
 
+# Kept in sync with the command line options in ci.py, so that metrics
+# are reported in the same language that the user uses.
+USER_FRIENDLY_PRODUCT_NAMES = {
+    out.Product(out.SAST()): "code",
+    out.Product(out.SCA()): "supply-chain",
+    out.Product(out.Secrets()): "secrets",
+}
```

Comment:
This seems pretty fundamental, maybe we should put it in `constants.py` instead?

Reply:
Done.